src/initializers/diamorphosis.ts

```diff
@@ -75,23 +75,9 @@ export default (config, orkaOptions: Partial<OrkaOptions>) => {
     maxBodyValueChars: undefined,
     ...config.riviere
   };
-  if (config.kafka) {
-    config.kafka.producer = {
-      brokers: [...(config.kafka.producer?.brokers || [])],
-      certificates: {
-        key: '',
-        cert: '',
-        ca: '',
-        ...config.kafka.producer?.certificates
-      },
-      sasl: {
-        username: '',
-        password: '',
-        ...config.kafka.producer?.sasl
-      },
-      ...config.kafka.producer
-    };
-  }
+
```

Comment:
I think that we had  the `if (config.kafka) {` to not initialize kafka if it was missing from config. Is this change going to initialize kafka even if it is not present in the config file?

Reply:
Since I initialise it as an empty string and brokers as an empty array those will still be evaluated to false.
My logic here is that on application level you won't  even need to set those variables on config. You could directly add them as env variables (in .env in development) and still work. Less configuration is better.

My idea is to do the same with the rest too, mongodb, rabbitMQ and redis. WDYT?

Reply:
Agree